x/pocketcore/types/cache.go

```diff
@@ -41,7 +42,8 @@ func (cs *CacheStorage) Init(dir, name string, dbType db.DBBackendType, maxEntri
 	var err error
 	cs.Cache, err = lru.New(maxEntries)
 	if err != nil {
-		panic(err)
+		fmt.Println(fmt.Errorf("could not initialize cache storage: " + err.Error()))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("could not initialize cache storage: %s", err.Error())
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)